--- /home/hinckley/lmp_3spn2/verbose/src/angle_hybrid.cpp.orig	2014-04-29 13:17:59.627937564 -0700
+++ ../angle_hybrid.cpp	2015-10-12 06:34:49.819507334 -0700
@@ -225,7 +225,7 @@
     keywords[nstyles] = new char[strlen(arg[i])+1];
     strcpy(keywords[nstyles],arg[i]);
     istyle = i;
-    if (strcmp(arg[i],"table") == 0) i++;
+    if ((strcmp(arg[i],"table") == 0) || (strcmp(arg[i],"list") == 0)) i++;
     i++;
     while (i < narg && !isalpha(arg[i][0])) i++;
     styles[nstyles]->settings(i-istyle-1,&arg[istyle+1]);
